docs: update mgt-todo documentation (#21549)
microsoftgraph/microsoft-graph-docs: concepts/toolkit/components/todo.md

## Authentication

The todo component uses the global authentication provider described in the [authentication documentation](../providers/providers.md).

## Cache

The `mgt-todo` component doesn't cache any data.

## Localization

> The control exposes the following variables that can be localized. See how you setup localization [here](../customize-components/localization.md).

Reviewer: ```suggestion
The control exposes the following variables that can be localized. For more information, see [setup localization](../customize-components/localization.md).
```

> 

Reviewer: @musale Is this the valid format for default values? Why do you want to keep them in quotes?

Author: Yes, this is the valid format for the default values, should be a back-tick though.
